src/EnergyPlus/HeatRecovery.cc

```diff
@@ -3291,25 +3291,20 @@ namespace HeatRecovery {
         using General::CreateSysTimeIntervalString;
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
+	auto &thisError = state.dataHeatRecovery->error1;
```

Comment:
This is purely a readability reference, and could have used `ErrorTracker2 &thisError =`

Reply:
Could have. But ErrorTracker2 is a struct and I am allowing `auto` for those elsewhere.